src/components/Select/Select.module.css

```diff
@@ -0,0 +1,115 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # SELECT
+\*------------------------------------*/
+
+/**
+ * Select field used to select one option from a list of options.
+ */
+.select {
+  position: relative;
+}
+
+/**
+ * Compact variant.
+ */
+.select--compact {
+  width: min-content;
+}
+.select-button--compact {
+  padding: var(--eds-size-half);
+}
+
+/**
+ * Label on top of the select trigger button to label the select field.
+ */
+.label {
+  @mixin eds-theme-typography-label-md;
+  color: var(--eds-theme-color-form-label);
+  margin-bottom: var(--eds-size-half);
+}
+
+/**
+ * Visually hides the label element if a aria-label is passed,
+ * but element is kept for screen reading accessibility.
+ */
+.u-is-vishidden {
+  @mixin visuallyHide;
+}
```

Comment:
Where is this being used?

Reply:
oh yah this was a carry-over from the dropdown component. I can remove this

Reply:
Thanks! I really wish we had a linter for this..